### ISSUE TITLE
Angular 5 error

### ISSUE THREAD

**Issue by skorunka** [matiboy/angular2-prettyjson#23]
Getting this error after upgrading my project to Angular 5:

```
compiler.js:466 Uncaught Error: Unexpected value 'PrettyJsonModule' imported by the module 'SharedModule'. Please add a @NgModule annotation.
    at syntaxError (compiler.js:466)
    at eval (compiler.js:15089)
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver.getNgModuleMetadata (compiler.js:15072)
    at CompileMetadataResolver.getNgModuleSummary (compiler.js:14999)
    at eval (compiler.js:15087)
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver.getNgModuleMetadata (compiler.js:15072)
    at JitCompiler._loadModules (compiler.js:33501)
    at JitCompiler._compileModuleAndComponents (compiler.js:33462)
```

Was working fine before the upgrade.

Is that a problem with the `angular2-prettyjson`?

Thank you.

**Comment by sam-higton** [comment]
I'm getting this issue too, but only when I compile for production. Did you find a solution?

**Comment by skorunka** [comment]
No, the solution was to stop using this library for now :(

**Comment by matiboy** [comment]
Hi @skorunka  and @sam-higton 

Sorry for not being more responsive; it should be fixed in 2.0.6

The issue should not reappear with new versions of Angular since I (finally) changed the dependency to `>=2.0.0` and leaving it up to npm/yarn to confirm that your application has something higher than 2. As explained [here](https://github.com/angular/angular/issues/15763#issuecomment-291812276)

Please let me know then I can close

Mat
